packages/core/src/associations/belongs-to-many.ts

```
@@ -908,6 +910,51 @@ Add your own primary key to the through model, on different attributes than the
+  /**
+   * Create multiple new instances of the associated model and associate them with this.
+   *
+   * @param sourceInstance source instance
+   * @param valuesArray array of values for target model instances
+   * @param options Options passed to bulkCreate and add
+   */
+  async createMultiple(
+    sourceInstance: SourceModel,
+    valuesArray: Array<CreationAttributes<TargetModel>>,
+    options: BelongsToManyCreateAssociationsMixinOptions<TargetModel> = {},
+  ): Promise<Array<TargetModel>> {
```

ESLint

[error] Array type using 'Array<TargetModel>' is forbidden for simple types. Use 'TargetModel[]' instead.

(@typescript-eslint/array-type)